plenum/server/consensus/ordering_service.py

```diff
@@ -211,7 +211,13 @@ def __init__(self,
         self._stasher.subscribe(PrePrepare, self.process_preprepare)
         self._stasher.subscribe(Prepare, self.process_prepare)
         self._stasher.subscribe(Commit, self.process_commit)
+        self._stasher.subscribe(ApplyNewView, self.process_apply_new_view)
         self._stasher.subscribe_to(network)
+        self._stasher.subscribe_to(self._bus)
```

Comment:
I think the current logic is correct because of 2 reasons:
1) I think we should have very strong distinction and separation between Internal and External messages. I think from the business logic point of view it should have no (or almost no) difference. Think about a service as an Actor receiving messages. It doesn't matter how the message was delivered. Moreover, depending on the configuration of actors/services, we may change message delivery (using of internal/external/inter process buses). 
2) I suspect that if we have different stashers for internal and external messages, then we may break the order of received messages (which is critical for unstash) and have the same  we had situation we had with different stasher instances created for Checkpointer and Ordering Services.